modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -10,6 +10,24 @@ Remember to xref:backup-restore/planning.adoc[plan your backup] carefully and ba
 Note that it is not allowed to take a backup of a database alias, only physical databases can be backed up.
 ====
 
+Before starting a backup, pay attention to your checkpointing policy, as the checkpoint has to be completed before the backup starts.
+
+Consider the following statements:
+
+* The backup cannot begin until the checkpoint finishes.
```

Comment:
Same here, a bit nitty, but I think it's important to state that the checkpointing is part of the backup job. So I'd say: the backup cannot proceed until the checkpoint finishes.